androidlibrary_lib/src/main/java/org/opendatakit/fragment/AlertDialogFragment.java

```diff
@@ -165,14 +168,17 @@ private void setTitle(String newTitle) {
       Fragment f = mgr.findFragmentById(fragmentId);
       setTargetFragment(f, RequestCodes.ALERT_DIALOG.ordinal());
 
-      AlertDialog dlg = new AlertDialog.Builder(getActivity())
-          .setIcon(android.R.drawable.ic_dialog_info).setTitle(title).setMessage(message)
-          .setCancelable(false).setPositiveButton(getString(R.string.ok), this).create();
-      dlg.setCanceledOnTouchOutside(false);
+      androidx.appcompat.app.AlertDialog alertDialog=new MaterialAlertDialogBuilder(getActivity(), R.style.Theme_MaterialComponents_Light_Dialog_Alert)
```

Comment:
Since you're using the AndroidX dialog here, the one in `getAlertDialog` of this class should also be changed. Otherwise there would be a crash on screen rotation.